tox.ini

```diff
@@ -16,12 +16,22 @@ deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/dev_requirements.txt
 
-[testenv:integration]
+[testenv:integration-spark-http]
```

Comment:
Tiny thing: I think we should update the `envlist` in line 3 above to reflect the fact that there are two different `integration` tests. I'm think `integration-spark-thrift` over `integration-spark-http`